go.mod

```diff
@@ -45,18 +45,18 @@ require (
 	github.com/opencontainers/runtime-spec v1.0.3-0.20200728170252-4d89ac9fbff6
 	github.com/opencontainers/selinux v1.8.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/client_golang v1.9.0
 	github.com/sirupsen/logrus v1.7.0
-	github.com/stretchr/testify v1.6.1
-	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
-	github.com/tchap/go-patricia v2.2.6+incompatible
-	github.com/urfave/cli v1.22.2
+	github.com/stretchr/testify v1.7.0
+	github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635
+	github.com/tchap/go-patricia v2.3.0+incompatible
+	github.com/urfave/cli v1.22.5
```

Comment:
Note that there's a regression in v1.22.2 and up; see https://github.com/opencontainers/runc/blob/cb269306807dcfd7f742c8444aca2e638bd81550/go.mod#L22-L23

Might affect `ctr` as well (if so, we may want to pin to the older version). I have a PR pending https://github.com/urfave/cli/pull/1135, but time got the best of me to revisit it